clients/telemetry/src/main/java/org/eclipse/hono/client/telemetry/TelemetrySender.java

```diff
@@ -12,7 +12,7 @@
  */
 
 
-package org.eclipse.hono.adapter.client.telemetry;
+package org.eclipse.hono.client.telemetry;
```

Comment:
How about updating copyright notice with current year?

Reply:
The same is also applicable to `ConnectionEventProducer` and `HonoEventConnectionEventProducerTest`.

Reply:
I have adapted the headers accordingly ...